.gitignore

```diff
@@ -37,3 +37,9 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# Backend settings
+backend/env/.env
+backend/env/.terraform/
+backend/src/AuditApi/local.settings.json
+backend/src/PatientTestsApi/local.settings.json
```

Comment:
are these two in the repo ?

Reply:
no, it's stuff i had left over from a different branch i was working on. i know it's kind of weird to include those ignores here but i didn't want to delete them

Reply:
understand, consider maybe git stash to keep it clean

Reply:
done